lab-taylor/server.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const net = require('net');
+const port = process.argv[2] || 3000;
+const ClientPool = require('./lib/client-pool');
+const wackPool = new ClientPool();
+
+const server = net.createServer();
+
+server.on('connection', (socket) => {
```

Comment:
Nothing is wrong with what you've written here, I thought I'd just mention that this could be made into a one line statement like so:

server.on('connection', (socket) => wackPool.emit('register', socket));

The difference between this version and yours is only that in this case wackPool.emit('register', socket) is additionally being returned (which is implicit in this kind of syntax).  Your version is more readable, but since it's a simple statement just thought I'd mention :)